app/models/publish_services_metadata_event.rb

```diff
@@ -5,19 +5,20 @@
 class PublishServicesMetadataEvent < Event
   include Utilities::Configuration::Settings
 
-  attr_reader :storage_key, :json_data
+  attr_reader :storage_key, :metadata
```

Comment:
`:storage_key` could be removed from here and the instance var on line 21 could be changed to a local var

Reply:
oh nice, well spotted. I just pushed a new commit